src/libcore/option.rs

```diff
@@ -188,10 +188,10 @@ impl<T> Option<T> {
     /// # Example
     ///
     /// ```
-    /// let x: Option<uint> = Some(2);
+    /// let x: Option<usize> = Some(2);
     /// assert_eq!(x.is_some(), true);
     ///
-    /// let x: Option<uint> = None;
+    /// let x: Option<usize> = None;
```

Comment:
Is there a reason to leave these as `usize` rather than `i32`? (for pedagogy against using usize randomly)